.gitignore

```diff
@@ -1 +1,15 @@
-.idea/
```

Comment:
.idea/ contains Jetbrains configuration files, so it must be included in .gitignore